screencastr.thor

```diff
@@ -35,6 +37,22 @@ class Screencastr < Thor
   desc "transcode VIDEO_PATH", "Transcode video file to mp4 format"
   def transcode(video_path)
```

Comment:
Should this method have options as an input that we merge in to a default options hash?

Reply:
Yes, I will definitely add those.